src/common/reconciler.go

```diff
@@ -112,27 +114,28 @@ func (r *ServiceReconciler) lookupService(registration opslevel_jq_parser.Servic
 		foundService, err := r.client.GetService(alias)
 		if err != nil {
 			gotError = err
-		} else {
+		} else if foundService != nil {
```

Comment:
Can this please be shuffled around to something like:
```go
} else if foundService == nil {
    log.Warn().Msgf("unexpected happened: got service with alias '%s' but the result is nil", alias)
} else if foundService.Id != "" {
   foundServices[string(foundService.Id)] = foundService
} else
  log.Warn().Msgf("unexpected happened: got service with alias '%s' but the result has no ID", alias)
}
```

Reply:
@davidbloss I re-ordered this if statement to be easier to read